src/main/scala/io/sdkman/changelogs/GcnMigrations.scala

```diff
@@ -0,0 +1,49 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "080")
+class MicronautMigrations {
```

Comment:
shouldn't this be called `GcnMigrations`?

Reply:
Indeed that was a missed change from using Micronaut as reference.